test/spec/modules/teadsBidAdapter_spec.js

```diff
@@ -0,0 +1,254 @@
+import {expect} from 'chai';
+import {spec} from 'modules/teadsBidAdapter';
+import {newBidder} from 'src/adapters/bidderFactory';
+
+const ENDPOINT = '//a.teads.tv/hb/bid-request';
+const AD_SCRIPT = '<script type="text/javascript" class="teads" async="true" src="http://a.teads.tv/hb/bid-request/hb/getAdSettings"></script>"';
+
+describe('teadsBidAdapter', () => {
```

Comment:
Can you change the arrow syntax in `describe` functions to use the normal syntax?  Ie change `, () => {` to `, function() {`

The normal syntax is more inline with Mocha, and most other adapter's test files now use this style.